domain/entities/transfers/transfers.go

```diff
@@ -0,0 +1,36 @@
+package transfers
+
+import (
+	"errors"
+	"time"
+)
+
+var (
+	errInvalidLengthID = errors.New("Empty ID not allowed")
```

Comment:
```suggestion
	ErrInvalidLengthID = errors.New("Empty ID not allowed")
```
O que acha de exportar esses erros? Assim as outras camadas conseguem acessar esses valores, checar qual tipo de erro é e tomar ações diferentes com base nessa informação.

Reply:
Corrigido.